bin-src/ui/messages/errors/gitOneCommit.ts

```diff
@@ -8,14 +8,20 @@ export default (isGithubAction = false) =>
   isGithubAction
     ? dedent(chalk`
       ${error} {bold Found only one commit}
-      This typically means you've checked out a shallow copy of the Git repository, which {bold actions/checkout@v2} does by default.
-      In order for Chromatic to correctly determine baseline commits, we need access to the full Git history graph.
-      With {bold actions/checkout@v2}, you can enable this by setting 'fetch-depth: 0'.
-      ${info} Read more at ${link('https://www.chromatic.com/docs/github-actions')}
+      This typically means you have ran into one of the following scenarios:
+      - You've checked out a shallow copy of the Git repository, which {bold actions/checkout@v2} does by default.
+        In order for Chromatic to correctly determine baseline commits, we need access to the full Git history graph.
+        With {bold actions/checkout@v2}, you can enable this by setting 'fetch-depth: 0'.
+        ${info} Read more at ${link('https://www.chromatic.com/docs/github-actions')}
+      - You've only made a single commit so far. 
+        Chromatic needs an additional commit to be able to detect what's changed. 
```

Comment:
@thafryer Could this be more explicit IE "Please make one more commit..." rather than "Chromatic needs..."? Maybe DX has an opinion but I think that's clearer.

Reply:
I agree with @alright-fine ! other than that it's good!

Reply:
Great suggestion! Made a tweak!